package.json

```diff
@@ -16,7 +16,8 @@
 		"lerna:deploy": "yarn build && lerna version --conventional-commits && lerna publish from-git",
 		"lint": "tsc --noEmit && eslint --ext js,ts,tsx .",
 		"start": "yarn workspace @regen-network/demo-app start",
-		"test": "lerna run test"
+		"test": "lerna run test",
+		"postinstall": "patch-package"
```

Comment:
cosmjs as a monorepo w/ sub packages required publishing each sub package individually, and all of them being forks w/ different names (e.g. `@cosmjs-regen/amino` instead of `@cosmjs/amino`.

We discussed this on our call this morning, and decided to stick with the patch approach (pending confirming that the patch gets run by regen-web correctly).